bin/satellite-clone.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import subprocess
+import sys
+
+def yes_or_no(question):
+    while "the answer is invalid":
+        reply = str(raw_input(question+' (y/n): ')).lower().strip()
+        if reply[0] == 'y':
+            return True
+        if reply[0] == 'n':
+            exit(1)
+
+def main():
+    parser = argparse.ArgumentParser(usage="Usage: satellite-clone [options]\n" \
```

Comment:
This currently renders as:
```
usage: Usage: satellite-clone [options]
Example: satellite-clone --start-at-task="Clean yum info"
```

If you don't pass `usage`, `argparse` autogenerates the following:
```
usage: satellite-clone.py [-h] [-v] [--start-at-task START_AT_TASK]
```

I don't really think the example is adding much here, and if we drop that, the autogenerated usage is just fine, IMHO.